diffsky/experimental/disk_bulge_modeling/generate_bulge_disk_sample.py

```diff
@@ -0,0 +1,93 @@
+import jax.numpy as jnp
+import numpy as np
+from diffsky.mc_diffsky import mc_diffstar_cenpop, mc_diffstar_galpop
```

Comment:
@evevkovacs watch out for global imports of diffsky from within diffsky. All imports should be relative. It looks like we will need to replace this call with a new call to diffstarpop.

Reply:
Regarding the code in `legacy`, the code is calling` _age_weights_from_params`,  but in [dsps.sfh.diffburst.py](https://github.com/ArgonneCPAC/dsps/blob/2cbd7d1beb5e510400261d4b3f0180725d7e82d4/dsps/sfh/diffburst.py) I see only [_pureburst_age_weights_from_params](https://github.com/ArgonneCPAC/dsps/blob/2cbd7d1beb5e510400261d4b3f0180725d7e82d4/dsps/sfh/diffburst.py#L112) or [_compute_bursty_age_weights_from_params](https://github.com/ArgonneCPAC/dsps/blob/2cbd7d1beb5e510400261d4b3f0180725d7e82d4/dsps/sfh/diffburst.py#L218). Do you know which one is the correct function to call?

Reply:
Yes, all the comments above have been addressed.
1. The relative imports are [fixed](https://github.com/ArgonneCPAC/diffsky/blob/e22637e1abfdb53917e612f3d4830d9402abbf6e/diffsky/experimental/disk_bulge_modeling/generate_bulge_disk_sample.py#L3)
2. The [imports](https://github.com/ArgonneCPAC/diffsky/blob/e22637e1abfdb53917e612f3d4830d9402abbf6e/diffsky/experimental/disk_bulge_modeling/disk_bulge_kernels.py#L14) from diffburst.py now come from dsps. I had to fix a bunch of code in the tests to get everything to work.